tests/c_languages/CPPCleanBearTest.py

```diff
@@ -17,6 +18,33 @@
 """
 
 
+test_include_paths_file = """
+#include "foo.h"
+#include "bar.h"
+
+int main() {
+    foo();
+    bar();
+    return 0;
+}
+"""
+
+
+def get_testdirectory_name(test_file):
+    return os.path.join(os.path.dirname(__file__), 'test_files', test_file)
+
+
 CPPCleanBearTest = verify_local_bear(CPPCleanBear,
                                      valid_files=(good_file,),
-                                     invalid_files=(bad_file,))
+                                     invalid_files=(bad_file,
+                                                    test_include_paths_file,),
+                                     tempfile_kwargs={'suffix': '.cpp'})
+
+CPPCleanBearValidIncludePathTest = verify_local_bear(
+    CPPCleanBear,
+    valid_files=(test_include_paths_file,),
+    invalid_files=(),
+    settings={'include_paths': get_testdirectory_name('headers1')+',' +
+              get_testdirectory_name('headers2')},
+    tempfile_kwargs={'suffix': '.cpp'}
```

Comment:
maybe you can also omit this here in this commit, and move it to the same commit when you introduce the suffix for the above test? So you have 1. commit: Add additional include support 2. commit: Use .cpp ending for tested files

Reply:
Tests fail when i dont add .cpp ending for files :P

Reply:
@Makman2  I could reverse the order of the steps tho :P First introduce .cpp ending and then add additional include support :)

Reply:
hm interesting why it fails though :3 but then you don't have to use 2 different commits, totally legit to keep this here :+1:

Reply:
@Makman2 Ok :) Could you have a look at the failed travis and circle builds?
Appear to have a common issue of sourceforge ssh connection not being possible. If you feel I could raise an issue for the same then :)